webknossos/webknossos/dataset/dataset.py

```diff
@@ -887,19 +894,19 @@ def add_layer_from_images(
                 module="webknossos",
             )
             if executor is None:
-                shapes_and_max_ids = [func_per_chunk(i) for i in args]
-            else:
-                # There are race-conditions about setting the bbox of the layer.
-                # The bbox is set correctly afterwards, ignore errors here:
-                warnings.filterwarnings(
-                    "ignore",
-                    message=".*properties were found on disk which are newer than the ones that were seen last time.*",
-                    category=UserWarning,
-                    module="webknossos",
-                )
-                shapes_and_max_ids = wait_and_ensure_success(
-                    executor.map_to_futures(func_per_chunk, args),
-                )
+                executor = get_executor_for_args(None)
```

Comment:
This uses multiprocessing now by default, right? Probably fine for now, but might create memory problems more easily.